lib/src/countly_flutter.dart

```diff
@@ -696,8 +704,10 @@ class Countly {
 
   /// Get currently used device Id type.
   /// Should be call after Countly init
+  @Deprecated('getDeviceIDType is deprecated, use getDeviceIDType of Countly.instance.deviceID instead')
   static Future<DeviceIdType?> getDeviceIDType() async {
     log('Calling "getDeviceIDType"');
+    log('getDeviceIDType is deprecated, use getDeviceIDType of Countly.instance.deviceID instead', logLevel: LogLevel.WARNING);
```

Comment:
Couldn't deprecated methods use new methods? Like this getDeviceIDType could call Countly.instance.deviceID?

Reply:
Our previous upgrades in flutter did not do this. e.g line 393. @turtledreams what do you think?